ops/charms/node_base.py

```diff
@@ -41,21 +48,34 @@ class NodeLabelError(Exception):
 
     def __init__(
         self,
-        charm: Charm,
+        charm: CharmImpl,
         kubeconfig_path: Union[PathLike, str],
+        *,
         kubectl: Optional[PathLike] = "/snap/bin/kubectl",
-    ):
+        user_label_key: str = "labels",
+        timeout: Optional[PositiveInt] = None,
+    ) -> None:
+        """Initialize the LabelMaker.
+
+        @param charm: The charm instance
+        @param kubeconfig_path: Path to the kubeconfig file
+        @param kubectl: Path to the kubectl binary
+        @param user_label_key: The key in the charm config where the user labels are stored
+        @param timeout: Number of seconds to retry a command
```

Comment:
I'm not familiar with this type of docstring. I thought we followed the Google Style Guide for [docstrings](https://google.github.io/styleguide/pyguide.html#381-docstrings).

Reply:
This is an "older" style of doc strings, We can change this library but i just used what was already in the library.  I don't mind adjusting if we think it reads better

Reply:
Let’s keep it as is, since we’d need to refactor all the docstrings for consistency. Thanks!

Reply:
it's too late  Copilot has already cashed the check i sent for adjusting them to the new style